tests/units/hindawi/cassettes/test_hindawi_get_file_with_wrong_DOI.yaml

```diff
@@ -26,17 +26,17 @@ interactions:
         Content-Type:
```

Comment:
It got changed because we are trying to make a request again if the error was raised. Before it was 3 times, no - 6

Reply:
Apparently 😅